src/main/java/org/pih/petl/job/type/PentahoJob.java

```diff
@@ -68,10 +67,10 @@ public void execute(final ExecutionContext context) throws Exception {
         System.setProperty("KETTLE_HOME", jobDir.getAbsolutePath());
 
         try {
-            // Configure kettle.properties with PIH_PENTAHO_HOME which we aquire from the configuration
-            File srcDir = appConfig.getPetlHomeDir();
+            // Configure kettle.properties with PIH_PENTAHO_HOME which we acquire from the configuration
+            File srcDir = new File(configuration.getProperty(PIH_PENTAHO_HOME, jobFile.getParent()));
```

Comment:
This was necessary to allow the pih-pentaho codebase to be loaded from someplace outside of the petl install directory, configurable by server as needed.